flixel/input/FlxPointer.hx

```diff
@@ -120,8 +120,8 @@ class FlxPointer
 	 */
 	public inline function setGlobalScreenPositionUnsafe(newX:Float, newY:Float):Void 
 	{
-		_globalScreenX = Std.int(newX / FlxG.scaleMode.scale.x);
-		_globalScreenY = Std.int(newY / FlxG.scaleMode.scale.y);
+		_globalScreenX = Std.int(FlxG.scaleMode.mouseMultiplier.x * newX / FlxG.scaleMode.scale.x);
```

Comment:
Since this is in `FlxPointer`, it affects touches as well. Because of that, `pointerMultiplier` is probably a better name than `mouseMultiplier`.